packages/caching/config/config.php

```diff
@@ -20,6 +20,7 @@
     $parameters = $containerConfigurator->parameters();
 
     $parameters->set(Option::ENABLE_CACHE, false);
+    $parameters->set(Option::CACHE_DIR, '_rector_cached_files');
```

Comment:
This looks like the root directory will contain the cache directory.

Reply:
Better now? Anyone can choose destination now.